static-assets/components/cstudio-forms/forms-engine.js

```diff
@@ -1898,6 +1898,17 @@ var CStudioForms =
                   saveFn(type === 'saveAndPreview', type !== 'saveAndClose', null, type);
                 };
 
+                document.addEventListener('keydown', (e) => {
+                  const isOSX = /mac/i.test(navigator.platform);
+                  if ((isOSX ? e.metaKey : e.ctrlKey) && e.key === 's') {
+                    // Prevent the Save dialog to open
+                    e.preventDefault();
+                    // Blur current focused element for the model to update before saving
+                    document.activeElement.blur();
+                    saveFn(false, true, null, 'save');
```

Comment:
@jvega190, does this call the currently selected function of the button (e.g. save as draft, save & close, save, etc)?

Reply:
it calls the save as draft function (doesn't close the form). Should it call the selected one?